docs/guides/runtime-cli-managing.md

```diff
@@ -1,204 +0,0 @@
-# Managing runtimes with the CLI
```

Comment:
why was this removed?

Reply:
There is no control cli anymore, we deprecated it.

Reply:
Guide updated.